index.js

```diff
@@ -58,6 +58,7 @@ function logger (opts) {
     function renderPerformance () {
       var entries = window.performance.getEntriesByName('choo:render')
       var index = entries.length - 1
+      if (index < 0) return
```

Comment:
This should probably be `return log.info('render')` so we still log the `'render'` event - we might drop it otherwise :(